lms/djangoapps/commerce/utils.py

```diff
@@ -32,3 +37,19 @@ def audit_log(name, **kwargs):
     message = u'{name}: {payload}'.format(name=name, payload=payload)
 
     log.info(message)
+
+
+class EcommerceService(object):
+    """ Helper class for ecommerce service integration. """
+    def __init__(self):
+        self.config = CommerceConfiguration.current()
+
+    def is_enabled(self):
+        """ Check if the service is enabled and that it's not a microsite. """
+        return self.config.checkout_on_ecommerce_service and not microsite.is_request_in_microsite()
```

Comment:
Yes, we will be eliminating "microsite" from the platform over the next 2-3 weeks, replacing it with theming and the Django Sites framework.  So any code references to microsite should be instead directed through theming.